tcl/wheredoesthisgo.tcl

```diff
@@ -28,17 +28,21 @@ proc open_file {filename} {
 # ------------------------------------------------------------------------------
 # procs for panels (openpanel, savepanel)
 
-proc pdtk_openpanel {target localdir} {
+proc pdtk_openpanel {target localdir mode} {
```

Comment:
from a compatibility pov, it probably makes sense to make the `mode` argument optional and default to `0`:

~~~tcl
proc pdtk_openpanel {target localdir {mode 0}} {
~~~